### PR TITLE
reduces error logs for client error scenarios

### PR DESCRIPTION
**Pull request by shamsimam**
## Changes proposed in this PR

- reduces error logs for client error scenarios
- removes the `suppress-logging` flag

## Why are we making these changes?

Avoids corrupting the log files with exception stack traces.

### CODE REVIEW

File: waiter/src/waiter/util/utils.clj [hunk 1]
```diff
@@ -256,18 +256,19 @@
...
       ex
-      (ex-info "Internal error" {:status 500} ex))))
+      (ex-info (str "Internal error: " (.getMessage ex)) {:status 500} ex))))
 
 (defn exception->response
   "Converts an exception into a ring response."
   [^Exception ex request]
   (let [wrapped-ex (wrap-unhandled-exception ex)
-        {:keys [friendly-error-message headers message status suppress-logging] :as data} (ex-data wrapped-ex)
+        {:keys [friendly-error-message headers logging message status] :as data} (ex-data wrapped-ex)
```
Comment (dposada) [inline waiter/src/waiter/util/utils.clj]: Maybe `log-level` instead of `logging`?

File: waiter/src/waiter/util/utils.clj [hunk 2]
```diff
@@ -256,18 +256,19 @@
...
         response-msg (if (or message friendly-error-message)
                        (str/trim (str message \newline friendly-error-message))
                        (.getMessage wrapped-ex))
         processed-headers (into {} (for [[k v] headers] [(name k) (str v)]))]
-    (when-not suppress-logging
+    (condp = logging
+      :info (log/info (.getMessage wrapped-ex))
```
Comment (dposada) [inline waiter/src/waiter/util/utils.clj]: Should we add `:warn` support?

File: waiter/src/waiter/util/utils.clj (hunk 1 above)
Comment (shamsimam) [reply waiter/src/waiter/util/utils.clj]: Done.

File: waiter/src/waiter/util/utils.clj (hunk 2 above)
Comment (shamsimam) [reply waiter/src/waiter/util/utils.clj]: Done.

**Pull request merged**